xml/ha_storage_protection.xml

```diff
@@ -879,12 +937,18 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
       about this step. Taken from FATE entry, developers could add more info.
      </remark>
      <para>
+      <remark>taroth 2018-04-28: I would list this as steps:
+       1. Configure a <property>on-fail=fence</property> property for a resource monitor operation: [...]
+          (not sure if giving the line is enough, perhaps also mention command with which to set this property?)
+       2. Produce a failure of the monitoring operation (how???).
```

Comment:
Hrm, quite depends on the resource.  Usually by killing the daemon of it's a service.

Reply:
Thanks, added this information.